types.go

```diff
@@ -196,6 +197,31 @@ func UnmarshalToByTypeURL(typeURL string, value []byte, out interface{}) error {
 	return err
 }
 
+// FromAny converts typeurl.Any to google.golang.org/protobuf/types/known/anypb.Any.
+func FromAny(from Any) *anypb.Any {
```

Comment:
Seems like this should be `ToProto`, we have way overused the word "Any" in this package. Also `interface{}` should be changed to `any` in a future PR 😆

Reply:
What about `MarshalAnyToProto` which was also moved in from containerd. `ToProto` and `MarshalAnyToProto`, kind of becomes confusing with the huge no.of other unmarshalling funcs. 


Yes the overuse of Any is  kind of confusing, and if we change the `interface{}` to `any` it  adds one more into the mix 🥲

Reply:
renamed to `MarshalAny` and `MarshalAnyToProto`